web-client/src/app/views/receive/receive.page.html

```diff
@@ -0,0 +1,33 @@
+<ion-header class="ion-no-border">
+  <app-header></app-header>
+</ion-header>
+
+<ion-content>
+  <div class="ion-text-center">
+    <h1 class="font-semibold font-nasalization">Receive</h1>
+  </div>
+  <ion-card color="white" class="mx-3 my-0">
+    <ion-card-header class="ion-text-center">
+      <ion-text class="text-2xl font-bold" color="tertiary"
+        >Scan to pay me
+      </ion-text>
+    </ion-card-header>
+    <ion-card-content>
+      <qrcode
+        [qrdata]="'Your data string'"
+        errorCorrectionLevel="H"
+        elementType="img"
+        [width]="1000"
+        [margin]="2"
+      ></qrcode>
+    </ion-card-content>
+  </ion-card>
+  <ion-list>
+    <app-action-item
+      *ngFor="let item of actionItems"
+      [preTitle]="item.preTitle"
```

Comment:
We just need to check the styling for the preTitle here. The contrast seems a bit low, not sure if that is how it is on the designs. See the UI Review in chromatic.